OTAnalytics/plugin_ui/customtkinter_gui/toplevel_flows.py

```diff
@@ -22,25 +24,37 @@ def __init__(
         super().__init__(**kwargs)
         self.title(title)
         self._section_ids = section_ids
+        self._current_id = tkinter.StringVar()
         self.input_values: dict = self.__create_input_values(input_values)
         self.protocol("WM_DELETE_WINDOW", self.close)
         self._initial_position = initial_position
+        self.__set_initial_values()
         self._get_widgets()
         self._place_widgets()
         self._set_initial_position()
         self._set_focus()
         self._set_close_on_return_key()
 
+    def __set_initial_values(self) -> None:
+        self._current_id.set(self.input_values.get(FLOW_ID, ""))
+
     def __create_input_values(self, input_values: Optional[dict]) -> dict:
         if input_values:
             return input_values
         return {
+            FLOW_ID: "",
             START_SECTION: "",
             END_SECTION: "",
             DISTANCE: 0,
         }
 
     def _get_widgets(self) -> None:
+        self.label_id = CTkLabel(master=self, text="Id")
```

Comment:
Should the user know, that he is entering the `ID`? For sections, I called it `name`...